### PR TITLE
Release version 5.0.1

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Release version 5.0.1
| Type?             | 
| BC breaks?        | 
| Deprecations?     | 
| Fixed ticket?     | 
| Sponsor company   |
| How to test?      |

### CODE REVIEW

**Comment by Hlavtox** [comment]
@PrestaShop/qa-functional One quick upgrade from 1.7.8 to 8.1 should be sufficient, not much changed.

**Review by ShaiMagal — DISMISSED** [review]

**Comment by matks** [comment]
Blocked by https://github.com/PrestaShop/autoupgrade/pull/671

File: upgrade/sql/8.1.4.sql
Comment (jolelievre) [inline upgrade/sql/8.1.4.sql:1]: I think this one should be renamed into `8.1.5.sql` first, the `8.1.4` was released separately as a security version

**Review by jolelievre — CHANGES_REQUESTED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by Hlavtox** [comment]
After CI is fixed in https://github.com/PrestaShop/autoupgrade/pull/673 we can go on

**Review by ShaiMagal — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @Hlavtox 

Thank you for your PR, I tested it and it seems to works as you can see : 

[recording(117).webm](https://github.com/PrestaShop/autoupgrade/assets/100699445/c69c1465-b954-46c8-a320-07d58b0b4a6b)

I tested to upgrade from 1.7.8.11 to 8.0.5 to 8.1.4
I do a rollboack to 8.0.5 and to 1.7.8.11 

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Pull request merged**